shell/platform/windows/angle_surface_manager.cc

```diff
@@ -81,7 +90,6 @@ bool AngleSurfaceManager::Initialize() {
                                default_display_attributes);
   if (egl_display_ == EGL_NO_DISPLAY) {
     std::cerr << "EGL: Failed to get a compatible EGLdisplay" << std::endl;
-    return false;
   }
 
   if (eglInitialize(egl_display_, nullptr, nullptr) == EGL_FALSE) {
```

Comment:
Is calling `eglInitialize` on `EGL_NO_DISPLAY` actually valid? It seems like this needs more restructuring than just removing the early return above.

Reply:
I believe it should be fine: https://www.khronos.org/registry/EGL/sdk/docs/man/html/eglInitialize.xhtml - `EGL_BAD_DISPLAY is generated if display is not an EGL display connection.`

Reply:
Actually, to test this manually I set `egl_display_` to `EGL_NO_DISPLAY` for the D3D11 attempts and it worked fine.